services/emailer/src/namex_emailer/services/helpers.py

```diff
@@ -59,13 +59,41 @@ def query_nr_number(identifier: str):
 
 
 @staticmethod
-def get_magic_link(nr_number, email, phone):
+def get_magic_link(nr_data):
     """Return a magic link."""
     BUSINESS_REGISTRY_URL = current_app.config.get("BUSINESS_REGISTRY_URL")
+    magic_link_route = {
+        'NEW': 'incorporateNow',
+        'MVE': 'continueInNow',
+        'AML': 'amalgamateNow',
+        # 'REN': 'registerNow'
```

Comment:
`REN` is for "renew".

"registerNow" is for NEW firm NRs (instead of coop or corp).

I'd like to see logic for NEW firm NRs, as we will likely add the magic link to their emails in the future.